server/src/main/java/com/cloud/api/query/QueryManagerImpl.java

```diff
@@ -3672,6 +3676,10 @@ else if (!template.isPublicTemplate() && caller.getType() != Account.ACCOUNT_TYP
             sc.addAnd("dataCenterId", SearchCriteria.Op.SC, zoneSc);
         }
 
+        if (templateTag != null && !templateTag.equalsIgnoreCase("")) {
```

Comment:
templateTag is unknown here, @ravening .

Reply:
@ravening could you please fix it ?

Reply:
@DaanHoogland can you explain why? As far as I see it is being passed as a parameter to `templateChecks()`

Reply:
it was a build error @nvazquez , If it's fixed, it's fixed.

Reply:
Copy, was checking the lastest changes so wasn't seeing why it was unknown, seems good now